src/app/components/Table/PercentChange.tsx

```diff
@@ -0,0 +1,21 @@
+import Image from "next/image";
+import { PercentChangeDiv } from "@/app/styling/components/Table/styled.PercentChange";
+
+interface Props {
+    percentChange: number;
+}
+
+const PercentChange = ({ percentChange }: Props) => {
+    const $isPositive = percentChange >= 0 ? true : false;
```

Comment:
const $isPositive = percentChange >= 0 

and why do you need the $

Reply:
It's a "transient prop", styled components will not pass props prefixed with "$" to the DOM [image](https://github.com/napqueenkaila/crypto-app/assets/125321591/7c7a2103-6580-4a52-93ea-dd899789491b)